src/main/java/seedu/address/model/transaction/Amount.java

```diff
@@ -29,6 +30,16 @@ public static boolean isValidAmount(String test) {
         }
     }
 
+    /**
+     * Adds {@code ammountToAdd} to the current amount.
+     *
+     * @return the total amount after adding.
+     */
+    public Amount add(Amount amountToAdd) {
+        double totalAmount = amountToAdd.amount + this.amount;
```

Comment:
[Suggestion]
Will it be better if you did this? :
double totalAmount;
totalAmount = ....;

Split them up?